dev-util/colm/colm-0.13.0.4-r1.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="COmputer Language Manipulation"
+HOMEPAGE="https://www.colm.net/open-source/colm/"
+SRC_URI="https://www.colm.net/files/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~x86"
+
+src_configure(){
```

Comment:
tiny nitpick, for consistency: `src_configure() {` (notice the space)

Reply:
done